src/pudl/settings.py

```diff
@@ -248,36 +248,24 @@ class Eia860Settings(GenericDatasetSettings):
 
     data_source: ClassVar[DataSource] = DataSource.from_id("eia860")
     years: list[int] = data_source.working_partitions["years"]
-
     eia860m: bool = True
-    eia860m_year_month: ClassVar[str] = max(
-        DataSource.from_id("eia860m").working_partitions["year_months"]
-    )
-
-    @field_validator("eia860m")
-    @classmethod
-    def check_eia860m_year_month(cls, eia860m: bool) -> bool:
-        """Check 860m date-year is exactly one year after most recent working 860 year.
-
-        Args:
-            eia860m: True if 860m is requested.
 
-        Returns:
-            eia860m: True if 860m is requested.
-
-        Raises:
-            ValueError: the 860m date is within 860 working years.
-        """
-        eia860m_year = pd.to_datetime(cls.eia860m_year_month).year
-        expected_year = max(cls.data_source.working_partitions["years"]) + 1
-        if eia860m and (eia860m_year != expected_year):
-            raise AssertionError(
-                f"Attempting to integrate an eia860m year "
-                f"({eia860m_year}) from {cls.eia860m_year_month} not immediately following"
-                f"the eia860 years: {cls.data_source.working_partitions['years']}. "
-                f"Consider switching eia860m parameter to False."
-            )
-        return eia860m
+    @property
+    def eia860m_year_month(self) -> list[str]:
```

Comment:
two things:
- because this can now be a list of `year_months` I would suggest making this plural!
- this is not necessary/blocking at all but it'd be nice to use the `eia860m` bool to set or not this guy. We use the bool in the extract step later but it'd be nice to just wipe the `year_months` if we are not doing 860m at all